lms/views/lti/oidc.py

```diff
@@ -56,11 +56,11 @@ def oidc_view(request):
     params = request.parsed_params
 
     registration = request.find_service(LTIRegistrationService).get(
-        issuer=params["iss"], client_id=params["client_id"]
+        issuer=params["iss"], client_id=params.get("client_id")
     )
     if not registration:
         raise HTTPForbidden(
-            f"""Registration not found. iss:'{params["iss"]}', client_id:'{params["client_id"]}'"""
+            f"""Registration not found. iss:'{params["iss"]}', client_id:'{params.get("client_id")}'"""
```

Comment:
Doing something like:

```issuer, client_id = params["iss"], params.get("client_id")```

Would prevent you from having to repeat the access here.